.circleci/config.yml

```diff
@@ -102,40 +102,35 @@ jobs:
     working_directory: ~/crux
     steps:
         - checkout
-        - attach_workspace:
-            at: ~/
 
-        - run: sh ./crux-bench/bin/download-dataset.sh
-        - run: lein sub -s crux-bench uberjar
-        - run:
-            command: |
-              echo 'export BUILD_ARG="--build-arg crux_commit=$CIRCLE_SHA1"' >> $BASH_ENV
-              echo 'export BUILD_TAGS="commit-$CIRCLE_SHA1"' >> $BASH_ENV
+        - run: md5sum **/project.clj > .circleci-cache-key
+        - restore_cache:
+            keys:
+              - v1-dependencies-{{ checksum ".circleci-cache-key" }}
+              # fallback to using the latest cache if no exact match is found
+              - v1-dependencies-
 
-        - setup_remote_docker
-        - aws-ecr/build-and-push-image:
-            dockerfile: 'Dockerfile'
-            extra-build-args: '${BUILD_ARG}'
-            path: "crux-bench/"
-            repo: 'crux-bench'
-            tag: '${BUILD_TAGS}'
-
-  deploy-master:
-    docker:
-      - image: circleci/clojure:lein-2.9.1
+        - run: md5sum **/crux-bench/bin/download-dataset.sh > .circleci-dataset-cache-key
```

Comment:
don't think you want `**/crux-bench` here - this is saying 'in all the subdirectories of this directory, look for a `crux-bench` directory'. You should be able to run `md5sum **/crux-bench/bin/download-dataset.sh` in the top-level of your local Crux repo - what happens?

Reply:
also, can this be 'bench-dataset', just to be unambiguous?

Reply:
I mostly did it based off of the `project.clj` md5 also being ran using ** - can change. Although, when running it my own local crux repo not much different happens than just running it normally? It does say the md5 followed by `checkouts/crux-bench/bin/download-dataset.sh`